src/lib.ts

```diff
@@ -53,15 +57,19 @@ export function generate(
     createCollectionEnum(sortedCollectionNames),
     ALIAS_TYPE_DEFINITIONS,
     BASE_SYSTEM_FIELDS_DEFINITION,
+    BASE_SYSTEM_CREATE_FIELDS_DEFINITION,
+    BASE_SYSTEM_UPDATE_FIELDS_DEFINITION,
     AUTH_SYSTEM_FIELDS_DEFINITION,
+    AUTH_SYSTEM_CREATE_FIELDS_DEFINITION,
+    AUTH_SYSTEM_UPDATE_FIELDS_DEFINITION,
     RECORD_TYPE_COMMENT,
     ...recordTypes,
     responseTypes.join("\n"),
     ALL_RECORD_RESPONSE_COMMENT,
     createCollectionRecords(sortedCollectionNames),
     createCollectionResponses(sortedCollectionNames),
     options.sdk && TYPED_POCKETBASE_COMMENT,
-    options.sdk && createTypedPocketbase(sortedCollectionNames),
+    UTILITY_TYPES,
```

Comment:
Move this up so it's under `createCollectionResponses`. I think `TypedPocketBase` should be the last thing in the file